### ISSUE TITLE
Program crashes when I try to synchronize my settings

### ISSUE THREAD

**Issue by SandroRDS** [microsoft/vscode#207966]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.87.2
- OS Version: Ubuntu 22.04.4 LTS

Steps to Reproduce:

1. Launch VSCode right after installing.
2. Click on the "Accounts" icon and go to the option "Turn on Setting Sync".

After that, the program triggers a prompt asking if I want to end the program or wait, but even though I ask to wait, it continues showing dialog boxes, as if it had entered an infinite loop.

**Comment by sandy081** [comment]
May I know at which step is it hanging? Is it possible to share a gif?

Can you please share following logs

- F1 > Open View... > Window
- F1 > Open View... > Settings Sync

**Comment by SandroRDS** [comment]
For some reason, today when I tried to reproduce it, it started working normally again! That's weird. Basically, when I clicked on the option to sync my settings, it opened the options bar for me to select to log in to a Microsoft or GitHub account. But at that moment, I couldn't do anything anymore, VSCode would crash and show the message that it had stopped working. As it just crashed and I needed to close the application, it didn't generate any logs here.
